### PR TITLE
May 2021 Corrections

### LINKED ISSUE #1345: Correction to Anthology ID 2021.nlp4if-1.15
**Please be sure to add the Anthology ID in the title**

[See here](https://www.aclweb.org/anthology/info/corrections/) to read about the three types of corrections.

## Revision or erratum: please add the following information**

- [ ] I have attached the revised PDF or erratum to this issue
- [x] I have written a brief description of the changes below

The brief description of the changes will be displayed on your paper page, and should be written in a neutral (third-person) fashion with complete sentences. For example:

> This revision corrects a typo in Equation 7. It also introduces a new figure, Figure 12, which plots the relationship between our proposed model and the baselines.

**Description of changes**
I would to like modify the Author names for the paper's acl-page. (Note the author names in the actual PDF paper are as wanted)
- Malak Abdullah Abdullah ------------->Malak Abdullah
- Ahmed Al-Qarqaz ---------------> Ahmed Qarqaz

### PR DESCRIPTION
**Pull request by xinru1414**

### CODE REVIEW

**Review by xinru1414 — COMMENTED** [review]
closes closes #1296

**Comment by kitsing** [comment]
Hi @xinru1414 , we have updated #1342 to also correct the abstract (in the metadata) . Can you please take a look? I can create a separate issue for that, if required.
Again thanks very much for your help! 🙏

**Comment by xinru1414** [comment]
> #1342

@kitsing can you confirm that the link to the pdf is updated?

**Comment by kitsing** [comment]
Yes, the link is good. It is only the metadata abstract (specifically [this line](https://github.com/acl-org/acl-anthology/blob/359c2859afdb4b4861c87092a8bf9b9b94b94420/data/xml/2021.naacl.xml#L4217)) that needs to be fixed.

Thank you!

**Comment by xinru1414** [comment]
> Yes, the link is good (and updated).
> 
> The metadata abstract that needs to be fixed is specifically [this line](https://github.com/acl-org/acl-anthology/blob/359c2859afdb4b4861c87092a8bf9b9b94b94420/data/xml/2021.naacl.xml#L4217).
> 
> Thank you!

@kitsing Fixed.

File: data/xml/2020.nlpcovid19.xml [hunk 1]
```diff
@@ -551,8 +551,11 @@
...
-      <url hash="f25a00f9">2020.nlpcovid19-2.24</url>
+      <url hash="2f7fda9c">2020.nlpcovid19-2.24</url>
       <doi>10.18653/v1/2020.nlpcovid19-2.24</doi>
+      <revision id="1" href="2020.nlpcovid19-2.24v1" hash="f25a00f9"/>
+      <revision id="2" href="2020.nlpcovid19-2.24v2" hash="74cce4ec" date="2021-06-03">changed section 5.3 and figure 3, added section 6</revision>
+      <revision id="3" href="2020.nlpcovid19-2.24v3" hash="2f7fda9c" date="2021-06-03">changed section 5.3 and figure 3, added section 6.</revision>
     </paper>
```
Comment (mjpost) [inline data/xml/2020.nlpcovid19.xml:559]: It looks like this one got added twice?

File: data/xml/2020.paclic.xml [hunk 2]
```diff
@@ -13,7 +13,9 @@
       <url hash="49fbe989">2020.paclic-1</url>
     </meta>
     <frontmatter>
-      <url hash="df2f405d">2020.paclic-1.0</url>
+      <url hash="eff2b951">2020.paclic-1.0</url>
+      <revision id="1" href="2020.paclic-1.0v1" hash="df2f405d"/>
+      <revision id="2" href="2020.paclic-1.0v2" hash="eff2b951" date="2021-06-03">Fontmatter correction</revision>
     </frontmatter>
```
Comment (mjpost) [inline data/xml/2020.paclic.xml:19]: Typo in "Frontmatter"

File: data/xml/2020.paclic.xml [hunk 3]
```diff
@@ -293,14 +295,18 @@
       <author><first>Ran</first><last>Tao</last></author>
       <author><first>Gang</first><last>Peng</last></author>
       <pages>286–293</pages>
-      <url hash="afd6ee5a">2020.paclic-1.33</url>
+      <url hash="21e2f21f">2020.paclic-1.33</url>
+      <revision id="1" href="2020.paclic-1.33v1" hash="afd6ee5a"/>
+      <revision id="2" href="2020.paclic-1.33v2" hash="21e2f21f" date="2021-06-03">2020.paclic-1.33 correction</revision>
     </paper>
```
Comment (mjpost) [inline data/xml/2020.paclic.xml:301]: Can you make this more informative? e.g., "Replaced incorrect version of paper" or whatever applies

File: data/xml/2020.paclic.xml [hunk 4]
```diff
@@ -293,14 +295,18 @@
...
     <paper id="34">
       <title>Construction of Associative Vocabulary Learning System for <fixed-case>J</fixed-case>apanese Learners</title>
       <author><first>Takehiro</first><last>Teraoka</last></author>
       <author><first>Tetsuo</first><last>Yamashita</last></author>
       <pages>294–301</pages>
-      <url hash="21e2f21f">2020.paclic-1.34</url>
+      <url hash="bc27f0f5">2020.paclic-1.34</url>
+      <revision id="1" href="2020.paclic-1.34v1" hash="21e2f21f"/>
+      <revision id="2" href="2020.paclic-1.34v2" hash="bc27f0f5" date="2021-06-03">2020.paclic-1.34 correction</revision>
     </paper>
```
Comment (mjpost) [inline data/xml/2020.paclic.xml:310]: Same

File: data/xml/2021.eacl.xml [hunk 5]
```diff
@@ -3083,6 +3083,8 @@
...
       <url hash="74cce4ec">2021.eacl-main.310</url>
       <attachment type="Software" hash="0b7f396a">2021.eacl-main.310.Software.zip</attachment>
+      <revision id="1" href="2021.eacl-main.310v1" hash="74cce4ec"/>
+      <revision id="2" href="2021.eacl-main.310v2" hash="74cce4ec" date="2021-06-03">dataset correction</revision>
     </paper>
```
Comment (mjpost) [inline data/xml/2021.eacl.xml:3088]: Please elaborate, make this a proper sentence (or sentence fragment).

File: data/xml/2021.naacl.xml [hunk 6]
```diff
@@ -4214,8 +4214,10 @@
...
+      <url hash="49e12cc6">2021.naacl-main.405</url>
+      <revision id="1" href="2021.naacl-main.405v1" hash="93f6fe2b"/>
+      <revision id="2" href="2021.naacl-main.405v2" hash="49e12cc6" date="2021-05-31">various corrections through the paper</revision>
     </paper>
```
Comment (mjpost) [inline data/xml/2021.naacl.xml:4221]: Capitalization

File: data/xml/2021.nodalida.xml [hunk 7]
```diff
@@ -448,7 +448,9 @@
       <author><first>Daniel</first><last>Varab</last></author>
       <pages>413–421</pages>
       <abstract>Danish language technology has been hindered by a lack of broad-coverage corpora at the scale modern NLP prefers. This paper describes the Danish Gigaword Corpus, the result of a focused effort to provide a diverse and freely-available one billion word corpus of Danish text. The Danish Gigaword corpus covers a wide array of time periods, domains, speakers’ socio-economic status, and Danish dialects.</abstract>
-      <url hash="ae5c62ef">2021.nodalida-main.46</url>
+      <url hash="d90398be">2021.nodalida-main.46</url>
+      <revision id="1" href="2021.nodalida-main.46v1" hash="ae5c62ef"/>
+      <revision id="2" href="2021.nodalida-main.46v2" hash="d90398be" date="2021-06-04">This revision amends an incorrect name in one of the cited words.</revision>
     </paper>
```
Comment (mjpost) [inline data/xml/2021.nodalida.xml:454]: words → works

File: data/xml/2021.teachingnlp.xml [hunk 8]
```diff
@@ -49,7 +49,9 @@
...
-      <url hash="87bed85c">2021.teachingnlp-1.3</url>
+      <url hash="cad60b21">2021.teachingnlp-1.3</url>
+      <revision id="1" href="2021.teachingnlp-1.3v1" hash="87bed85c"/>
+      <revision id="2" href="2021.teachingnlp-1.3v2" hash="cad60b21" date="2021-06-01">fixed typo in equation and added new figure</revision>
     </paper>
```
Comment (mjpost) [inline data/xml/2021.teachingnlp.xml:55]: caps

File: data/xml/P18.xml [hunk 9]
```diff
@@ -2760,9 +2760,11 @@
...
-      <url hash="277ef3b5">P18-1217</url>
+      <url hash="53aada76">P18-1217</url>
       <attachment type="poster" hash="5de089e9">P18-1217.Poster.pdf</attachment>
       <doi>10.18653/v1/P18-1217</doi>
+      <revision id="1" href="P18-1217v1" hash="277ef3b5"/>
+      <revision id="2" href="P18-1217v2" hash="53aada76" date="2021-06-01">added corresponding author</revision>
     </paper>
```
Comment (mjpost) [inline data/xml/P18.xml:2768]: capitalization

**Review by mjpost — COMMENTED** [review]
Mostly looks good! I added a bunch of comments regarding the formatting of revision notes.

File: data/xml/2020.nlpcovid19.xml (hunk 1 above)
Comment (xinru1414) [reply data/xml/2020.nlpcovid19.xml:559]: Yes, because I made a mistake linking the new pdf the first time

Comment (mjpost) [reply data/xml/2020.nlpcovid19.xml:559]: Okay—we should delete r3 and correct r2 manually, then. Or just reset the XML here to the original and re-process the revision.

**Review by mjpost — APPROVED** [review]
LGTM

**Pull request merged**